test/integration/steps/test_parent_steps.py

```diff
@@ -190,6 +190,20 @@ def a_change_request_is_valid():
     return context
 
 
+@given(parsers.parse('the field "{field}" is set to "{value}"'), target_fixture="context")
+def generic_event_config(context, field: str, value: str):
+    if field[0:7] == "Contact":
+        # If the change is a website then change the website entry, otherwise change the telephone one
+        if value[0:4] == "http":
```

Comment:
This statement suggests that all web entries from Changed events are prefixed with 'http'. Has this been verified

Reply:
In the test suite we don't currently have any tests that aren't prefixed with http. If we add them, then the code can be changed to
`if value[0:4] == "http" or value[0:3] == "www" ` as it stands this isn't necessary for the suite.